python/gui/auto_generated/qgsadvanceddigitizingdockwidget.sip.in

```diff
@@ -538,6 +538,12 @@ Convenient method to get the M value from the line edit wiget
 .. versionadded:: 3.22
 %End
 
+    CadCapacities capacities() const;
+%Docstring
+Returns the capacities
+
+.. versionadded:: 3.24
```

Comment:
```suggestion
.. versionadded:: 3.26
```

3.24 is now the LTR and unfortunately, it's not acceptable, for me, to back port this.